package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "interface-ipld-format",
+  "version": "0.0.2",
+  "description": "IPLD format",
+  "main": "index.js",
+  "types": "./types.d.ts",
+  "files": [
+    "types.d.ts"
+  ],
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ipld/interface-ipld-format.git"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
better fix this, `"(Apache-2.0 AND MIT)"`

Reply:
Done!